api/src/test/java/org/openmrs/module/metadatamapping/api/MetadataMappingServiceTest.java

```diff
@@ -786,9 +786,9 @@ public void getMetadataItem_shouldReturnUnretiredMetadataItemForUnretiredTerm()
 	}
 	
 	@Test
-	@Verifies(value = "not return retired metadata item for unretired term", method = "getMetadataItem(Class, String, "
+	@Verifies(value = "return retired metadata item for unretired term", method = "getMetadataItem(Class, String, "
 	        + "String)")
-	public void getMetadataItem_shouldNotReturnRetiredMetadataItemForUnretiredTerm() {
+	public void getMetadataItem_shouldReturnRetiredMetadataItemForUnretiredTerm() {
```

Comment:
Please make sure the `Verifies` annotation value matches the test method name and also the `verifies` tags in the javadoc for `org.openmrs.module.metadatamapping.api.MetadataMappingService#getMetadataItem`.